src/main/java/com/dongxuexidu/douban4j/provider/OAuthDoubanProvider.java

```diff
@@ -148,7 +148,7 @@ public AccessToken tradeAccessTokenWithCode (String code) throws DoubanException
       Map<String,String> params = new HashMap<String, String>();
       params.put("client_id", this.apiKey);
       params.put("client_secret", this.secretKey);
-      params.put("redirect_uri", DefaultConfigs.ACCESS_TOKEN_REDIRECT_URL);
+      params.put("redirect_uri",  this.redirectUrl);
```

Comment:
Yes, but this is in tradeAccessTokenWithCode method right? It will directly pass a 'null' value into a server through line 154 if you didnt check it here.